CHANGELOG.md

```diff
@@ -0,0 +1,20 @@
+# v0.2.0
```

Comment:
Title says release v0.1.1, but this says v0.2.0. Which one is right?

Reply:
Updated title. While I was writing the changelog, I realized that it can't be a patch release because besides fixing a bug, we also have new functionality.